mypy/typeshed/stdlib/types.pyi

```diff
@@ -602,11 +602,11 @@ if sys.version_info >= (3, 9):
         def __getattr__(self, name: str) -> Any: ...
 
 if sys.version_info >= (3, 10):
-    # This annotation allows `NoneType` to be used as a value of `type[None]`
-    NoneType: Final[type[None]] = ...
+    # ideally, this would have a `: TypeAlias`, but it makes mypy output
+    #  an invalid error on this line
+    NoneType = type(None)
```

Comment:
can it use the new type alias syntax instead

Reply:
no, the real isn't a type alias